python/ray/tune/trial.py

```diff
@@ -12,7 +12,10 @@
 
 from ray.tune import TuneError
 from ray.tune.logger import NoopLogger, UnifiedLogger, pretty_print
-from ray.tune.registry import _default_registry, get_registry, TRAINABLE_CLASS
+# NOTE(rkn): We import ray.tune.registry here instead of importing the names we
+# need because there are cyclic imports that may cause specific names to not
+# have been defined yet. See https://github.com/ray-project/ray/issues/1716.
+import ray.tune.registry
```

Comment:
what about `import ray.tune.registry as registry`

Reply:
I like full names for clarity, but I'm happy to change it if you want.